### DISCUSSION TITLE
2024 Core Maintainer Elections

### DISCUSSION THREAD

**Discussion by TheCodeTraveler** [CommunityToolkit/Maui#2011] [General]
# Core Maintainer Elections 2024 

Greetings fellow Open Source developers! 

We are seeking new members to the .NET MAUI Community Toolkit Core Maintainer team, and we want YOU, the community, to decide who joins us. On the Community Toolkit team, we strongly believe that core maintainers should come from the community and be elected by the community. 

Going forward, we will hold yearly elections each year in July, finalizing the election results at the [2024 August Monthly Standup](https://github.com/CommunityToolkit/Maui/wiki/Monthly-Standups).

This year, 2024, we are seeking two (2) additional Core Maintainers.

### Nominations

All candidates are self-nominated.

Before nominating yourself, please read through our [Core Team Responsibilities](https://github.com/CommunityToolkit/Maui/wiki/Core-Team-Tools-and-Responsibilities#responsibilities) to ensure you are able to commit to the team for a minimum of one (1) year, are able to attend our monthly standup, and are able to contribute a minimum of one activity per week. 

To nominate yourself, write a top-level comment below in this Discussion introducing yourself to the community and highlighting why you'd make the best core maintainer!

Your self-nomination should include, but is not limited to, the following details about yourself:
- Years of open-source experience
- Years of .NET MAUI experience (can also include Xamarin experience)
- Links to contributions, bug reports, discussions (etc.) to the [CommunityToolkit organization](https://github.com/CommunityToolkit) 
  - Can include contributions to the [Xamarin.CommunityToolkit](https://github.com/xamarin/xamarincommunitytoolkit)
  - Can include contributions to the [CommunityToolkit.Maui.Markup](https://github.com/communitytoolkit/maui.markup)
- Links to other open-source projects you maintain
- New Proposals + Bug Fixes you will contribute to the .NET MAUI Community Toolkits
- Proficiency in English + Other languages 
  - The .NET MAUI Community Toolkit conducts all Issues, Pull Requests, Discussions, Standups, etc in English
  - A multi-lingual core maintainers is a HUGE bonus given the vast diversity of our community
    - Our current team covers Dutch, Portuguese, Spanish, Ukrainian and Korean

And similar to the [StackOverflow Moderator Elections](https://stackoverflow.com/election/14 ), we seek core core maintainers who exemplify the following traits:

- Patient and fair
- Leads by example
- Shows respect for their fellow community members in their actions and words
- Open to some light but firm moderation to keep the community on track, and resolve uncommon disputes and exceptions

The nominees who receive the most upvotes will join the .NET MAUI Community Toolkit Core Maintainer Team.

### Voting 

Voting is simple! [Upvote the nomination](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#upvoting-a-comment) for each Core Maintainer whom you wish to join the team. 

To vote for a candidate, read through the nominations, below, and [Upvote the nomination comment](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#upvoting-a-comment) for the candidate whom you'd like to join the team.

You are allows to vote (upvote) as many nominees as you wish.

At the end of the 2024 election, the nominations with the top two (2) total votes will join the team.

We ask the community to avoid abusing the voting system by creating new GitHub accounts to vote additional times for a preferred candidate. We reserve the right to ban users from the Community Toolkit organization whom we suspect of creating multiple accounts to vote for a specific candidate in the yearly elections.

### Results

Results will be finalized and posted here during the [2024 August Monthly Standup](https://github.com/CommunityToolkit/Maui/wiki/Monthly-Standups).

**Comment by ne0rrmatrix** [comment]
Hello everyone. My name is James Crutchley. I have been involved almost exclusively in the dotnet Maui Community toolkit with a heavy focus on Media Element. I have almost all of my time free and feel I can continue to contribute to the project with more PR's for both bug fixes and feature requests/improvements. I am happy to listen to others and work with just about anyone. I am looking forward to hearing what the community thinks. I have had fun the last year or so working with the toolkit on personal projects and then on the toolkit fixing bugs and adding features. I have learned a lot about how to use c# and have found the community very welcoming.

**Reply by GeorgeLeithead** [reply]
Congratulations on becoming a Core Maintainer.  Well deserved.

**Comment by GeorgeLeithead** [comment]
Can a link to this be added to the WiKi page, under the 'Core Maintainers' heading, otherwise it's not that easy to find.

**Reply by GeorgeLeithead** [reply]
@brminnick Thoughts?

**Reply by VladislavAntonyuk** [reply]
Updated https://github.com/CommunityToolkit/Maui/wiki/Yearly-Elections

**Reply by GeorgeLeithead** [reply]
Thanks Vlad.  Much appreciated.

**Comment by CliffAgius** [comment]
Closing as the elections are over and it was covered on the [August Stand-up](https://github.com/CommunityToolkit/Maui/wiki/Monthly-Standups)
